gui/wxpython/core/giface.py

```diff
@@ -232,8 +232,13 @@ def __init__(self):
         # Signal emitted when workspace is changed
         self.workspaceChanged = Signal("StandaloneGrassInterface.workspaceChanged")
 
-        # Signal emitted when history should be updated
-        self.updateHistory = Signal("StandaloneGrassInterface.updateHistory")
+        # Signal emitted when entry to history is added
+        self.addEntryToHistory = Signal("StandaloneGrassInterface.addEntryToHistory")
```

Comment:
Should this be named "entryToHistoryAdded"  (or "historyEntryAdded") and the same for removing?